packages/lib-content/package.json

```diff
@@ -43,7 +43,7 @@
     "@zooniverse/grommet-theme": "3.x.x",
     "grommet": "2.x.x",
     "grommet-icons": "4.x.x",
-    "next": "~13.5.5",
+    "next": "~14.2.26",
```

Comment:
🤔 Why was this peer dependency stuck at 13.5.5?

Reply:
Good question. tl;dr - I'm not sure, but I think this is ok. 

I think the previous https://github.com/zooniverse/front-end-monorepo/pull/6434/files only bumped minor `next` versions, so maybe lib-content didn't get included. The last major `next` bump for lib-user was included individually - https://github.com/zooniverse/front-end-monorepo/pull/6367/files. I don't see any issues with lib-content.